requirements/prod.txt

```diff
@@ -20,8 +20,6 @@ click==7.1.2
     #   snowfakery
 colorama==0.4.4
     # via rich
-coloredlogs==15.0.1
-    # via -r requirements/prod.in
```

Comment:
@Br4nd0R Note for next MetaCI update: MetaCI uses coloredlogs (https://github.com/SFDO-Tooling/MetaCI/blob/922df3c24d9d8661c50a241dde7e4b580a823cd0/metaci/cumulusci/logger.py#L8) but doesn't explicitly include it as a dependency. Now that it won't get installed as as transitive dependency of cumulusci, MetaCI will either need to explicitly require it (in prod.in) or stop using it.